src/ExerciseRunner/CgiRunner.php

```diff
@@ -153,7 +160,11 @@ private function executePhpFile($fileName, RequestInterface $request, $type)
             $output = "HTTP/1.0 200 OK\r\n" . $output;
         }
 
-        return ResponseSerializer::fromString($output);
+        return (
```

Comment:
I'm not keen on the mix of return types here, `array` vs `string` is there a way to make it return just one type, always `array` maybe with an empty string or something?